src/hardware/motor.cpp

```diff
@@ -292,6 +294,28 @@ void StepperMotor::setMicrostepping(uint16_t setMicrostepping) {
 
         // Fix the microsteps per rotation
         this -> microstepsPerRotation = round(360.0 / microstepAngle);
+
+        // Set the microstepMultiplier according to the microstepDivisor
```

Comment:
This isn't how the microstep multiplier is supposed to work. I'll leave it in, but I'm going to add it as a config option.